# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../samba/sasl2.patch
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- samba-4.2.0/source4/auth/gensec/cyrus_sasl.c.vanilla	2015-03-30 10:10:46.044000000 +0000
+++ samba-4.2.0/source4/auth/gensec/cyrus_sasl.c	2015-03-30 10:10:51.684000000 +0000
@@ -26,7 +26,7 @@
 #include "auth/gensec/gensec_internal.h"
 #include "auth/gensec/gensec_proto.h"
 #include "auth/gensec/gensec_toplevel_proto.h"
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 
 NTSTATUS gensec_sasl_init(void);
 
